config-ui/src/models/Connection.js

```diff
@@ -35,7 +35,7 @@
  * @property {string|object?} plugin
  * @property {string|object?} provider
  * @property {string?} providerId
- * @property {<Array<DataEntity>>} entities
+ * @property {<Array[DataDomain]>} dataDomains
```

Comment:
Is this valid JSDoc format for specifying array of type?
```suggestion
 * @property {<Array<DataDomain>>} dataDomains
```

Reply:
fixed